package.json

```diff
@@ -38,6 +38,7 @@
     "@hapi/joi": "^16.1.8",
     "@medic/translation-checker": "^1.0.1",
     "@parcel/watcher": "^2.0.5",
+    "@xmldom/xmldom": "^0.8.2",
```

Comment:
FYI, `@xmldom/xmldom` is just the [new version](https://github.com/xmldom/xmldom/issues/271) of `xmldom`.